packages/babel-types/src/index.js

```diff
@@ -398,6 +404,7 @@ export function inherits(child, parent) {
   }
 
   t.inheritsComments(child, parent);
+  traverse.copyCache(parent, child);
```

Comment:
@amasad Probably way too late at this point, but do you happen to remember what issues you ran into when this wasn't here? I asked when you originally posted the PR and your comment was:

> Not just a performance win leaving this will cause some tests to fail because the node is orphaned and doesn't have an associated nodepath, so it has to create one and by doing so it will not have the objects that were passed at the time of the ast creation (namely hub and file etc) so it will also lead to a failing test

but as far as I can see, commenting out this line on `master` at the moment causes no failures at all. Were you potentially talking about other tests in your own plugins or something?